core/components/molecules/pagination/pagination.md

```diff
@@ -0,0 +1,36 @@
+```meta
+  category: Layout
+  description: "Paginate resources with a handy pagination component"
+```
+
+```jsx
+<Pagination items={20372} perPage={10} page={3} />
```

Comment:
To make the prop table work, you need to pass `{props}` to the component and you can set the intended defaults with the `defaults` prop

Example:

```jsx
< Pagination {props} defaults={{items: 20372, perPage: 10, page: 3}} />
```


More here: https://auth0-cosmos-master.now.sh/docs/#/contribution-guide